components/script/dom/bindings/reflector.rs

```diff
@@ -50,6 +51,18 @@ impl Reflector {
         self.object.handle()
     }
 
+    /// Get the JS runtime.
+    #[inline]
+    pub fn get_runtime(&self) -> *mut JSRuntime {
+        unsafe { JS_GetObjectRuntime(self.object.get()) }
+    }
+
+    /// Get the JS context.
+    #[inline]
+    pub fn get_cx(&self) -> *mut JSContext {
+        unsafe { JS_GetContext(self.get_runtime()) }
+    }
```

Comment:
This probably isn't needed any more, originally I was fighting to try to get Servo to have globals that aren't `GlobalScope`s but eventually gave up. This looks like code that was left over from that fight.